api/middleware/auth.js

```diff
@@ -11,7 +11,7 @@ const validateToken = async (req, res, next) => {
     next();
   } catch (err) {
     console.error(err);
-    response.status(401).json({ message: 'Invalid Access Token' });
+    response.status(401).json({ message: "Invalid Access Token" });
```

Comment:
해당 코드는 http status code를 보이기 위함으로 보입니다.
에러 `status`를 출력하기 위해 아래와 같이 수정해 주세요
```suggestion
err.statusCode = 400;
```